mayastor-test/e2e/setup/README.md

```diff
@@ -0,0 +1,38 @@
+
+This directory contains the scripts to bring up and environment ready for testing
+(for example by CI).
+
+
+## Pre-requisites
+
+1. Install libvirt plugin
+```
+# From https://github.com/vagrant-libvirt/vagrant-libvirt#installation
+#apt-get build-dep vagrant ruby-libvirt
+sudo apt-get install -y qemu libvirt-daemon-system libvirt-clients ebtables dnsmasq-base ruby-libvirt
+sudo apt-get install -y libxslt-dev libxml2-dev libvirt-dev zlib1g-dev ruby-dev
+vagrant plugin install vagrant-libvirt
```

Comment:
would be cool to have a nix-shell file in e2e directory and everyone who would run it would get all needed dependencies for running the e2e tests.

Reply:
I'll look at it outside of this PR